### ISSUE TITLE
Kubernetes v1.22

### ISSUE THREAD

**Issue by arcastro** [wunderio/csi-rclone#24]
I created a new cluster with 1.22, and tried the csi-rclone plugin

`csi-controller-rclone` fails to start with the following errors

```
Mon, Jan 17 2022 4:12:37 pm | I0118 00:12:37.340838 1 reflector.go:161] Listing and watching *v1beta1.VolumeAttachment from k8s.io/client-go/informers/factory.go:133
Mon, Jan 17 2022 4:12:37 pm | E0118 00:12:37.363637 1 reflector.go:126] k8s.io/client-go/informers/factory.go:133: Failed to list *v1beta1.VolumeAttachment: the server could not find the requested resource
```

Issue seems similar to the one here (separate project):
* https://github.com/hetznercloud/csi-driver/issues/220

Which I think was resolved by:
* https://github.com/hetznercloud/csi-driver/commit/fb905752233bb63fdf1790405c121d019c5b3dd5#diff-f546476e5fffe4b34bbb6a6d1e87e121e10b7b64df747cbe1560c4e1844dfe29

EDIT: I can confirm the issue is resolved by applying similar changes as the diff above. Tested and worked. I'll send a pull-request if I get a chance, but I'm not sure if these changes would break the currently working `1.13.x - 1.20.x` kubernetes versions. TBH, I have no idea how any of this works :) 

In the meantime, here's a couple screenshots of the necessary changes in case it helps anyone
* https://i.imgur.com/r41wHXy.png
* https://i.imgur.com/8Sz6T5u.png
* https://i.imgur.com/RzjsTyl.png
* https://i.imgur.com/0BanbVD.png

Apologies for not going the extra step of submitting a pull-request. Invested too much time on this already and screenshots are quick haha

**Comment by zer0def** [comment]
Should be adddressed by #23.

**Comment by arcastro** [comment]
> Should be addressed by #23.

@zer0def , I wish I'd seen that sooner haha. Thank you!
